atomics/include/desul/atomics/Fetch_Op_ScopeCaller.hpp

```diff
@@ -0,0 +1,12 @@
+/*
+Copyright (c) 2019, Lawrence Livermore National Security, LLC
+and DESUL project contributors. See the COPYRIGHT file for details.
+Source: https://github.com/desul/desul
+
+SPDX-License-Identifier: (BSD-3-Clause)
+*/
+
+#ifndef DESUL_ATOMICS_FECH_OP_SCOPECALLER_HPP_
+#define DESUL_ATOMICS_FECH_OP_SCOPECALLER_HPP_
+
```

Comment:
Is this file vestigial?

Reply:
No, it was my thinking we could have shortcuts something like
```C++
template <class T>
T atomic_fetch_add(T* ptr, T val, MEMORY_ORDER, MemoryScopeCaller) {
  T ret{*ptr};
  *ptr += val
  return ret;
}
```
but I ran out of steam.  It can be removed and we will just re-introduce later if appropriate.

Reply:
Ok, maybe a FIXME or an issue so we can remember the intent?

Reply:
Done